cocos/ui/UIRichText.cpp

```diff
@@ -1795,19 +1827,24 @@ void RichText::formatRenderers()
             float maxHeight = 0.0f;
             for (auto& iter : row)
             {
-                maxHeight = MAX(iter->getContentSize().height, maxHeight);
+                maxHeight = std::max(iter->getContentSize().height, maxHeight);
+            }
+            if (row.empty())
+            {
+                maxHeight = std::max(fontSize, _defaultHeights[i]);
```

Comment:
why height compare with font size?

Reply:
This is for continuous new lines.
eg："abc\n\n123" should display like this:
abc

123